chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
why was this file modified? accidental change from the dependency pr?

Reply:
It looks like the PR that changed finalizers didn't get generated YAML. This is the output of `make manifests`. I can remove, if desired.

Reply:
ah ok no worries, let's keep it